### PR TITLE
delete unused verifier code

### PR DESCRIPTION
**Pull request by phbnf**
Followup from #308

### CODE REVIEW

File: cmd/conformance/gcp/main.go [hunk 1]
```diff
@@ -37,7 +37,6 @@ var (
 	listen            = flag.String("listen", ":2024", "Address:port to listen on")
 	spanner           = flag.String("spanner", "", "Spanner resource URI ('projects/.../...')")
 	signer            = flag.String("signer", "", "Note signer to use to sign checkpoints")
-	verifier          = flag.String("verifier", "", "Note verifier corresponding to --signer")
```
Comment (mhutchinson) [inline cmd/conformance/gcp/main.go:40]: You'll need to update (and possibly deploy) `deployment/modules/gcp/conformance/main.tf`

File: deployment/modules/gcp/cloudbuild/main.tf [hunk 2]
```diff
@@ -94,7 +94,6 @@ resource "google_cloudbuild_trigger" "docker" {
       name   = "alpine/terragrunt"
       script = <<EOT
         export TESSERA_SIGNER=$(cat /workspace/key.sec)
```
Comment (phbnf) [inline deployment/modules/gcp/cloudbuild/main.tf:96]: Hmm, I think this can go as well.

File: cmd/conformance/gcp/main.go (hunk 1 above)
Comment (phbnf) [reply cmd/conformance/gcp/main.go:40]: Done - I think we should be good without deploying, right? Cloudbuild should just pick everything up?

Comment (mhutchinson) [reply cmd/conformance/gcp/main.go:40]: @AlCutter for the right answer - I can't recall which way this is deployed right now.

**Review by mhutchinson — APPROVED** [review]

File: cmd/conformance/gcp/main.go (hunk 1 above)
Comment (AlCutter) [reply cmd/conformance/gcp/main.go:40]: So long as this config always matches the flags in the binary I think it should be good as the conformance test is currently [re-]created from scratch each time.

**Review by AlCutter — APPROVED** [review]

**Pull request merged**